templates/pages/about.html

```diff
@@ -8,27 +8,53 @@
 
 
 	<div class="jumbotron">
-		<p>We, CodeCatz, are learning to code, beause we can and because we want to! </p>
-		<p>CodeCatz is a group of programming kittens and several slightly more experienced cats. Kittens want to learn coding, cats are here to help them. We started with Python and web applications, we'll continue with whatnots.</p>
-		<p> Who knows where we will end! </p>
+		<h2 class="about">About CodeCatz</h2>
+		<p class="intro">CodeCatz is a coding study group where women are always in majority. The group is dedicated all women who want to learn coding, however men are welcomed too.  Cats pass their coding knowledge to kittens through various group or personal projects and when the kittens are more skilled they pass their gained knowledge to new beginners. The objects of CodeCats is to assist women to enter in to the world of coding.</p>
+		<p class="intro">CodeCatz have only <strong>two rules</strong> for the members: <strong>chocolate</strong> at the meetings and <strong>full dedication to coding</strong>.</p>
+		<p class="intro">We are always open for a new potential members after, so come over to say hi, see how we work. Especially welcome if you're a female kitten, who wants to learn more about programming or an expert cat, who would like to help said kittens and learn new things from other Catz.</p>
+		<p class="intro">CodeCatz welcome also mentors or companies who see future of the women in programming and would like to assist out kittens in any shape or forms and change a world. </p>
+		<p>We code together <strong>every Wednesday</strong> at <strong>RAMPA Laboratorij, Kersnikova ulica 4, Ljubljana at 17:30</strong>.
+		<p class="intro"><strong>Follow TrackCat:</strong><p>
```

Comment:
this line is missing a closing p tag as well... instead there is an opening p tag :wink: